bin/match_file

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+require 'match'
+
+# CLI to encrypt/decrypt files using fastlane match encryption layer
+
+def usage
+  puts("USAGE: [encrypt|decrypt] input_path [output_path]")
+  exit(-1)
+end
+
+if ARGV.count < 2 || ARGV.count > 3
+  usage
+end
+
+method_name = ARGV.shift
+unless ['encrypt', 'decrypt'].include?(method_name)
+  usage
+end
+
+input_file = ARGV.shift
+
+if ARGV.count > 0
+  output_file = ARGV.shift
+else
+  output_file = input_file
+end
+
+def ask_password(msg)
+  ask(msg) do |q|
+    q.whitespace = :chomp
+    q.echo = "*"
+  end
+end
+
+def ask_password_twice
+  password = ask_password("Enter the password:")
+  other = ask_password("Enter the password again:")
+  if other == password
+    return password
+  else
+    return nil
+  end
+end
+
+# read the password
+password = nil
+loop do
+  password = ask_password_twice
+  break unless password.nil?
```

Comment:
Won't people get stuck in this loop without being able to quit until they enter 2 identical strings?

I'm having issues with my env currently, I couldn't test this 😢 but I remember some of fastlane's built-in "ask for X" prompts doesn't allow `ctrl+C` to quit 😬

Reply:
yes. Let's try to fix the lack of ctrl+C support first then ;)

Reply:
added. But just in match-file. Maybe overkill?

Reply:
in the end I removed the ctrl-C support as it doesn't seem to be possible to do it on older rubies.

Reply:
Nice!